From: uazo <uazo@users.noreply.github.com>
Date: Mon, 3 Mar 2025 12:04:53 +0000
Subject: Do not check for the presence of use_remoteexec

---
 build/toolchain/rbe.gni | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/build/toolchain/rbe.gni b/build/toolchain/rbe.gni
--- a/build/toolchain/rbe.gni
+++ b/build/toolchain/rbe.gni
@@ -4,11 +4,7 @@
 
 # Defines the configuration of Remote Build Execution (RBE).
 
-use_remoteexec_default =
-    exec_script(rebase_path("//build/toolchain/use_remoteexec_value.py"),
-                [],
-                "value",
-                [])
+use_remoteexec_default = false
 
 declare_args() {
   # Deprecated: Please use reclient_bin_dir instead.
--
