From: uazo <uazo@users.noreply.github.com>
Date: Thu, 15 May 2025 06:55:56 +0000
Subject: Try to fix 2082

---
 .../android_webview/nonembedded/NetworkFetcherTask.java         | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/android_webview/nonembedded/java/src/org/chromium/android_webview/nonembedded/NetworkFetcherTask.java b/android_webview/nonembedded/java/src/org/chromium/android_webview/nonembedded/NetworkFetcherTask.java
--- a/android_webview/nonembedded/java/src/org/chromium/android_webview/nonembedded/NetworkFetcherTask.java
+++ b/android_webview/nonembedded/java/src/org/chromium/android_webview/nonembedded/NetworkFetcherTask.java
@@ -248,7 +248,7 @@ public class NetworkFetcherTask {
                             xCupServerProof,
                             xRetryAfter);
 
-        } catch (IOException exception) {
+        } catch (Throwable exception) {
             Log.w(TAG, "IOException during post request.", exception);
             NetworkFetcherTaskJni.get()
                     .callPostRequestCompleteCallback(
--
